cranelift/frontend/src/frontend.rs

```diff
@@ -162,6 +162,12 @@ impl<'short, 'long> InstBuilderBase<'short> for FuncInstBuilder<'short, 'long> {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+/// This is the error variant returned from [`FunctionBuilder::try_use_var`].
+pub enum UseVariableError {
```

Comment:
This should also impl [Error](https://doc.rust-lang.org/std/error/trait.Error.html) and Display so that callers can easily use ? on it.